doc/changes/changes_1.0.1.md

```diff
@@ -0,0 +1,17 @@
+# DB2 Virtual Schemas 1.0.1, released 2021-??-??
```

Comment:
I think it should be a new major version, because we're breaking the interface by removing SQL_DIALECT